substrate/frame/election-provider-multi-block/src/unsigned/tests.rs

```diff
@@ -273,51 +273,6 @@ mod miner {
 		});
 	}
 
-	#[test]
```

Comment:
not very relevant, we should test the case that the signed/unsigned phases pass without a solution and that's it.